src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -234,7 +241,7 @@ IFACEMETHODIMP_(void) FancyZones::VirtualDesktopInitialize() noexcept
 // IFancyZonesCallback
 IFACEMETHODIMP_(void) FancyZones::WindowCreated(HWND window) noexcept
 {
-    if (m_settings->GetSettings().appLastZone_moveWindows)
+    if (IsInterestingWindow(window) && m_settings->GetSettings().appLastZone_moveWindows)

```

Comment:
It's better to first check for `m_settings->GetSettings().appLastZone_moveWindows`